WordPress/create-block-theme
# Add option to create blank theme

This PR is an idea for addressing #68 — an option for generating an "empty" block theme inside the current site's themes directory.

The boilerplate is pretty much [block-canvas](https://github.com/Automattic/themes/tree/trunk/block-canvas). I thought it would be best to put the boilerplate code inside the plugin so it could all be maintained in one place.

**To test**
- Check out the PR
- Select the last option, give the theme a name
- Verify you can activate the theme and edit it via the site editor
